### ISSUE TITLE
Comeback Ratchet!

### ISSUE THREAD

**Issue by kyawkyawsoezhu** [twbs/ratchet#867]
I have a web project only target to mobile users, kind of PWA. So I don't need mobile-first or responsive CSS library. 

since yesterday I try to find simple css/js library that target only mobile browser, surprisingly I couldn’t find even a single one, most library are for both mobile and desktop. I found some mobile only framework like framewrok7, ionic, onsen.io but they are full featured framework that might overkill me.

after spent several hours on google finally I found Ratchet, I'm very glad because it match my need, the exact library I want, but sadly there are some small bugs and the last commit is 2 years ago.

## My point is Ratchet is unique library on the market and I believe many developers have situation to need Ratchet

**Comment by leechenhwa** [comment]
@kyawkyawsoezhu you may see https://github.com/sdc-alibaba/SUI-Mobile which based on ratchet and f7.

**Comment by kyawkyawsoezhu** [comment]
@leechenhwa thanks for your suggestion, it look cool but I can't read Mandrin and i think the library not active anymore :sob: .

**Comment by ack202** [comment]
Thats true! I use Ratchet with Laravel to develop PWAs. I guess there are many other developers out there looking for exactly this.

**Comment by moeenio** [comment]
I second this. Would love this to come back to develop with Cordova.

**Comment by daudfauzy98** [comment]
Please continue this project 😭
